game.py

```diff
@@ -20,7 +20,30 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+    wrong_guesses_count = 0
+    
+    while wrong_guesses_count < SNOWMAN_MAX_WRONG_GUESSES:
+        print_snowman_graphic(wrong_guesses_count)
+        print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+        print("Wrong guesses so far:", " ".join(wrong_guesses_list))
```

Comment:
Really nice use of the `join` method here! If we just used a statement like `print(wrong_guesses_list)`, we would have ended up with a list that includes the brackets before and after (`[ ]`) and quotes around each letter. Your approach gives your program a more natural flow and look.